tt_eager/tt_lib/csrc/operations/primary/module.hpp

```diff
@@ -496,6 +497,25 @@ void py_module(py::module& m_primary) {
         py::arg("strategy").noconvert() = MorehSoftmaxBackwardOpParallelizationStrategy::NONE,
         py::arg("output_mem_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG,
         "Performs a softmin backward operation. Returns a input grad tensor.");
+
+    m_primary.def(
+        "moreh_arange",
+        &moreh_arange,
+        py::arg("start"),
+        py::arg("end"),
+        py::arg("step"),
+        py::arg("any").noconvert(),
+        py::arg("output_mem_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG,
+        "Performs a arange operation. Returns a output tensor.");
```

Comment:
```suggestion
        "Performs an arange operation. Returns an output tensor.");
```

Reply:
I updated to "Performs an arange operation. Returns an output tensor."